python/singa/image_tool.py

```diff
@@ -384,7 +388,7 @@ def crop8(self, patch, num_case=1, inplace=True):
         if num_case == patch5 + patch3:
             count = patch5
         else:
-            sample_list = range(0, patch5 + patch3)
+            sample_list = list(range(0, patch5 + patch3))
```

Comment:
`from past.builtins import range` can not be used because it is for` backward compatible` when converting from Python 3 to Python 2. 

`from builtins import range` is already used in this file. It adds a list for range:

http://python-future.org/compatible_idioms.html#range